src/default/assets/js/src/typedoc/components/MenuHighlight.ts

```diff
@@ -73,18 +73,20 @@ namespace typedoc
                 base = base.substr(0, base.indexOf('#'));
             }
 
-            this.$el.find('a').each((_index, el: Element) => {
-                var href = (el as HTMLAnchorElement).href;
+            const el: HTMLElement = this.el;
+            el.querySelectorAll('a').forEach(el => {
+                var href = el.href;
                 if (href.indexOf('#') == -1) return;
                 if (href.substr(0, base.length) != base) return;
 
                 var hash = href.substr(href.indexOf('#') + 1);
-                var $anchor = $('a.tsd-anchor[name=' + hash + ']');
-                if ($anchor.length == 0) return;
+                var anchor = document.querySelector<HTMLElement>('a.tsd-anchor[name=' + hash + ']');
+                var link = el.parentNode;
+                if (!anchor || !link) return;
```

Comment:
Looks like with this change the IAnchorInfo interface can have optionality removed from its members.

Reply:
I have removed this code
```ts
this.anchors = [{
  position: 0
}];
```
And I have used `-1` value for index, if we are on top of the page and we have not highlight menu item.